model/src/main/proto/analytics.proto

```diff
@@ -8,13 +8,13 @@ option java_multiple_files = true;
 // Supported event actions for analytics logging
 enum EventAction{
   // The undefined user action of an event
-  UNKNOWN_EVENT_ACTION = 0;
+  UNKNOWN_EVENT_ACTION_UNSPECIFIED = 0;
```

Comment:
Let this be EVENT_ACTION_UNSPECIFIED. UNKNOWN and UNSPECIFIED mean the same thing. Is buf asking you to write both UNKNOWN and UNSPECIFIED?

Reply:
No, it's passing, just checked

Reply:
Looks good. Resolving this on behalf of @vinitamurthi